### ISSUE TITLE
Training towards maximal policy loss

### ISSUE THREAD

**Issue by Akababa** [glinscott/leela-chess#432]
In self-play, instead of using various temperature schemes to decide the next move played, the client could choose the move which incurs maximal crossentropy loss. I think that perhaps this could help train the policy faster by "immunizing" the network against its blunders, effectively seeking out adversarial examples in self-play.

Any thoughts?

**Comment by gcp** [comment]
Unless I'm misunderstanding something, you want it to constantly play the biggest blunder possible?

I'm not sure why you expect a positive result from this. Not only will it confuse the value network further, that move is simply less likely to be good (=> considerably better than its policy score) than any other choice.

**Comment by Akababa** [comment]
Not the biggest blunder possible (bad wording), the one with largest cross-entropy distance between predicted score and playout score (fraction of visits), so it should still pick one of the top few moves.
